cron.py

```diff
@@ -21,20 +21,50 @@ def check_cron(schedule: str, current: datetime.datetime) -> bool:
     
     minute, hour, day_of_month, month, day_of_week = parts
     
-    if not _check_field(minute, current.minute, 0, 59, enable_slash=True):
+    if not _check_field(minute, current.minute, 0, 59):
         return False
     
-    if not _check_field(hour, current.hour, 0, 23, enable_slash=True):
+    if not _check_field(hour, current.hour, 0, 23):
         return False
     
-    if not _check_field(day_of_month, current.day, 1, 31, enable_slash=True):
+    if not _check_field(day_of_month, current.day, 1, 31):
         return False
     
-    if not _check_field(month, current.month, 1, 12, enable_slash=True):
+    if not _check_field(month, current.month, 1, 12):
         return False
     
     if day_of_week == '*':
         pass  # Wildcard matches any day
+    elif ',' in day_of_week:
+        days = day_of_week.split(',')
+        match_found = False
+        for day in days:
+            if day == '*':
+                match_found = True
+                break
+            elif day.upper() in ['MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN']:
+                day_map = {
+                    'MON': 0,
+                    'TUE': 1,
+                    'WED': 2,
+                    'THU': 3,
+                    'FRI': 4,
+                    'SAT': 5,
+                    'SUN': 6,
+                }
+                if current.weekday() == day_map[day.upper()]:
+                    match_found = True
+                    break
+            else:
+                try:
+                    int(day)
+                    raise ValueError(f"Day of week must be a three-letter abbreviation (MON, TUE, etc.), not a number: {day}")
+                except ValueError as e:
+                    if "Day of week must be" in str(e):
+                        raise
+                    raise ValueError(f"Invalid day of week: {day}. Must be a three-letter abbreviation (MON, TUE, etc.) or *")
+        if not match_found:
+            return False
```

Comment:
There's a bunch of duplicated code with the below `elif`. Add a helper function and dedup it